doc/releases/changelog-0.23.0.md

```diff
@@ -608,16 +629,26 @@ the `decimals` and `show_matrices` keywords are added. `qml.drawer.tape_text(tap
   ```
   [(#2276)](https://github.com/PennyLaneAI/pennylane/pull/2276)
```

Comment:
this PR link should be moved to the end of the line with the bullet point!

Reply:
Also I think we can convert the two codeblocks above into a single `pycon` code block